include/test/battle.h

```diff
@@ -663,6 +725,17 @@ extern struct BattleTestRunnerState *gBattleTestRunnerState;
 #define MEMBERS_7(a, b, c, d, e, f, g) a; b; c; d; e; f; g;
 #define MEMBERS_8(a, b, c, d, e, f, g, h) a; b; c; d; e; f; g; h;
 
+#define APPEND(...) VARARG_8(APPEND_, __VA_ARGS__)
+#define APPEND_0()
+#define APPEND_1(a) a,
+#define APPEND_2(a, b) a,  b,
+#define APPEND_3(a, b, c) a,  b,  c,
+#define APPEND_4(a, b, c, d) a,  b,  c,  d,
+#define APPEND_5(a, b, c, d, e) a,  b,  c,  d,  e,
+#define APPEND_6(a, b, c, d, e, f) a,  b,  c,  d,  e,  f,
+#define APPEND_7(a, b, c, d, e, f, g) a,  b,  c,  d,  e,  f,  g,
+#define APPEND_8(a, b, c, d, e, f, g, h) a,  b,  c,  d,  e,  f,  g,  h,
+
```

Comment:
It seems like `APPEND(...)` is equivalent to just `...`, so this shouldn't be needed.
```suggestion
```